system.go

```diff
@@ -0,0 +1,23 @@
+package utils
+
+import (
+	"time"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+// StopEtcd stops etcd on a specific host
+func StopEtcd(node TestbedNode) error {
```

Comment:
I need to expand on them some more. They are IterateNodes compatible 
functions. I’d like to add utility functions for consul, 
netplugin/netmaster (but in the netplugin test suite) and a few other 
things.

We can sit on this PR, but I don’t know how much time I’ll have to 
work on it this week, so we can merge it now and I can iterate in small 
chunks.

Alternatively, if you don’t think this belongs here, happy to move it 
back to volplugin.

On 11 Oct 2015, at 9:57, Madhav Puri wrote:

>> @@ -0,0 +1,23 @@
>> +package utils
>> +
>> +import (
>> +	"time"
>> +
>> +	log "github.com/Sirupsen/logrus"
>> +)
>> +
>> +// StopEtcd stops etcd on a specific host
>> +func StopEtcd(node TestbedNode) error {
>
> @erikh what is the usecase for these start/stop utilities?
>
> As of now we start them as part of Vagrantfile (with appropriate 
> arguments set). And usually there was never a need to kill these 
> during systemtests as we use cleanup scripts for cleaning up any 
> testcase related state. Do you want something similar?
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/contiv/systemtests-utils/pull/3/files#r41712770